webapp/components/Comment/Comment.vue

```diff
@@ -67,6 +76,7 @@ import ContentViewer from '~/components/Editor/ContentViewer'
 import CommentForm from '~/components/CommentForm/CommentForm'
 import CommentMutations from '~/graphql/CommentMutations'
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-01-09T08:14:33Z" title="Thursday, January 9th 2020, 9:14:33 am +01:00">Jan 9, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -79,6 +86,7 @@ export default {
       isTarget,
       isCollapsed: !isTarget,
       openEditCommentMenu: false,
+      answered: this.$t('post.comment.answered'),
```


Okay, now I understand.
Please use `this.$t('post.comment.answered‘)` directly in `:title="this.$t('post.comment.answered')“` otherwise it seems not to be responsive if you declare it as `data`.

Please rename the variable to `answer`, because it is what you do right now and not what you have done already …

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-01-10T07:22:13Z" title="Friday, January 10th 2020, 8:22:13 am +01:00">Jan 10, 2020</time>_

---


ok

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-10T19:18:43Z" title="Friday, January 10th 2020, 8:18:43 pm +01:00">Jan 10, 2020</time>_

---


@ogerly I think `reply` would fit here.

Reply:
> [<img alt="ogerly" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/ogerly) **Authored by [ogerly](https://github.com/ogerly)**
_<time datetime="2020-01-14T17:40:27Z" title="Tuesday, January 14th 2020, 6:40:27 pm +01:00">Jan 14, 2020</time>_

---


ok